runelite-client/src/main/java/net/runelite/client/ui/overlay/components/PanelComponent.java

```diff
@@ -44,8 +44,8 @@
 public class PanelComponent implements RenderableEntity
 {
 	private static final int TOP_BORDER = 4;
-	private static final int LEFT_BORDER = 4;
-	private static final int RIGHT_BORDER = 4;
+	public static final int LEFT_BORDER = 4;
+	public static final int RIGHT_BORDER = 4;
```

Comment:
Having to do this indicates you are doing something wrong.

Reply:
What is this "something"?

Reply:
Doing the line breaks outside, instead of letting it be done by TextComponent